edx_event_bus_kafka/internal/tests/test_consumer.py

```diff
@@ -433,9 +445,12 @@ def test_check_event_error(self):
     @ddt.data(True, False)
     def test_emit_success(self, audit_logging, mock_logger, mock_set_attribute):
         self.signal.disconnect(fake_receiver_raises_error)  # just successes for this one!
+        msg = copy.copy(self.normal_message)
```

Comment:
Is the copy necessary? setUp should be creating a fresh copy for each test.

(I'm fairly sure I was the one who introduced `copy` in the existing unit tests, but now I think it was actually unnecessary...)

Reply:
for the sake of keeping this PR clean I'll keep the legacy ones but fix the new ones